ResponsiveMedia/Drivers/ResponsiveMediaFieldDisplayDriver.cs

```diff
@@ -39,12 +39,15 @@ public ResponsiveMediaFieldDisplayDriver(IMediaFileStore mediaFileStore, IString
 
         public override IDisplayResult Display(ResponsiveMediaField field, BuildFieldDisplayContext context)
         {
+            var settings = context.PartFieldDefinition.Settings.ToObject<ResponsiveMediaFieldSettings>();
+            var data = field.HasData ? field.Data : settings.DefaultData;
```

Comment:
Wouldn't this mean the default would keep coming back if the user was trying to have no items in the field? When I've been implementing default values I've generally made them only apply on the initial create.

Reply:
I guess the downside to the approach you've taken is if I want to change the default image for all the existing content items, I'd have to go through each of the existing content items and change the image manually to the new default image.

The idea for this approach is to prevent there ever not being an image. Maybe "Default" is the wrong term, "Fallback" could be more appropriate?

Reply:
Oh I actually misread this too, I thought this was happening in the `Edit` get.

But yeah now you've pointed that out I agree 'Fallback' might be a better name for this setting.

Reply:
And now that I understand what this actually does I'm wondering if highlighting to the editor that there's a fallback image (and maybe showing it?) would be worthwhile?